package.json

```diff
@@ -25,6 +25,7 @@
     "@react-navigation/native": "^6.0.2",
     "@react-navigation/native-stack": "^6.1.0",
     "@react-navigation/stack": "^6.0.11",
+    "accordion-collapse-react-native": "^1.1.0",
```

Comment:
You don't seem to be actually using this dependency. Please remove it and update the `yarn.lock`.

Reply:
ok